app/models/donations/subscription.rb

```diff
@@ -1,7 +1,9 @@
 class Donations::Subscription < ApplicationRecord
   belongs_to :user
 
-  scope :active, -> { where(active: true) }
+  scope :active, -> { where(status: :active) }
+
+  enum status: { canceled: 0, overdue: 1, active: 2 }, _suffix: true
```

Comment:
I believe if you remove the `_suffix` here, you can remove the scope above. 

Then you get:
`Donations::Subscription.active` and `subscription.active?`

Reply:
I only use the suffix where it reads more clearly that way (e.g. if we wanted to have `subscription.active_status?` for some reason because gramatically it works better)